packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/NotificationsDropdown/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { Translation, Notifications } from '@suite-components';
+import { Dropdown, DropdownRef, variables } from '@trezor/components';
+import React, { useRef, useCallback, useState } from 'react';
+import styled from 'styled-components';
+import ActionItem from '../ActionItem';
+import { useActions } from '@suite-hooks';
+import * as notificationActions from '@suite-actions/notificationActions';
+
+const Wrapper = styled.div`
+    display: flex;
+    z-index: 10;
+    /* same as left padding for ActionItem */
+    margin-right: 28px;
```

Comment:
I'd try to have this configurable somehow (prop, const somewhere in a config or similar), because the comment might be overlooked :/